modules/apps/remote-app/remote-app-client-js/src/main/resources/META-INF/resources/js/types/index.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+type JSONValue =
```

Comment:
Typescript in DXP.. A new era begins

Reply:
![rainbow](https://user-images.githubusercontent.com/905006/90538309-4946f900-e176-11ea-9480-1ddced47dfc6.gif)

Reply:
![](https://media.giphy.com/media/ZdCJ4G17h7lIsrYfLs/giphy.gif)